ozaria/site/styles/play/level/tome/spell-palette-view.sass

```diff
@@ -0,0 +1,249 @@
+@import "app/styles/mixins"
+@import "app/styles/bootstrap/variables"
+@import "app/styles/play/variables"
+
+#spell-palette-view
+  position: absolute
+  right: $spell-pallete-right
+  width: $api-bar-width
+  bottom: 0px
+  top: 0px
+  padding: 0 4px 0px 10px
+  background: transparent url(/images/level/wood_texture.png)
+  background-size: auto 100%
+  z-index: 7
+  @include transition(top 0.25s ease-in-out, width 0.25s ease-in-out)
+  box-shadow: 10px 4px 4px black
+  overflow: hidden
+  #spell-palette-api-bar[disabled=disabled]
+    pointer-events: none
+    #palette-tab-api-area
+      opacity: 0.6
+  #level-view.real-time &, #level-view.cinematic &
+    display: none
+    visibility: hidden
+  &.open
+    width: $api-bar-width + 360px
+  .container
+    width: $api-bar-width + 360px
+    height: 100%
+    position: relative
+    .left
+      position: absolute
+      left: 0
+      top: 80px
+      bottom: 0px
+      height: auto
+      width: $api-bar-width - 13
+      .section-header, .sub-section-header
+        display: block
+        color: white
+        margin-right: 10px
+
+      .spell-palette-thang-entry-view
+        display: inline-block
+        border: 5px solid black
+        border-image: url(/images/level/thang_avatar_frame.png) 20 20 20 20 fill round
+        margin: 1px
+
+        img
+          background-color: black
+          width: 72px
+          height: 72px
+          border: 1px solid transparent
+          cursor: pointer
+        &:hover img
+          border: 1px solid #f3cd90
+        &.selected img
+          border: 1px solid white
+          cursor: auto
+
+      .nano-slider
+        background-color: rgba(243, 169, 49, 0.5)
+    .right
+      position: absolute
+      right: 20px
+      min-width: 200px
+      left: $api-bar-width - 10px
+      top: 60px
+      bottom: 0px
+      //width: calc(100% - #{$api-bar-width + 50px})
+      @include box-shadow(0 0 0 #000)
+      .scrollArea
+        max-height: calc(100% - 20px)
+        width: 100%
+        border-style: solid
+        border-image: url(/images/level/popover_border_background.png) 16 12 fill round
+        border-width: 16px 12px
+        margin-top: 20px
+        margin-bottom: 20px
+        position: relative
+        overflow-y: auto
+
+      .always-scroll-y
+        overflow-y: scroll
+        .description
+          img
+            float: left
+
+      .closeBtn
+        position: absolute;
```

Comment:
Could these semicolons be causing issues. Should we correct this going forward in this new file?

Reply:
Not sure, since I just copied it from the legacy folder. But thanks for noticing this, I will update this.